sdk/go/common/resource/plugin/analyzer_plugin_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2025, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugin
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/pulumi/pulumi/sdk/v3/go/common/resource/config"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/testing/diagtest"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAnalyzerSpawn(t *testing.T) {
+	if runtime.GOOS != "linux" {
+		t.Skip("Skipping test on non-linux platforms")
+	}
```

Comment:
Why?  A comment with the reason would probably be a good idea here.

Reply:
Sure, was just because I didn't want to write a matching powershell script but given how small it is maybe it's worth just adding

Reply:
I'm also fine skipping it, just want to make sure we understand why.  But also without a powershell script we should still be able to run it on MacOS?

Reply:
Added a batch script so this should run on everything now